<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1049</width>
    <height>816</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>631</width>
      <height>441</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="button_browse_files">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>700</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse File</string>
    </property>
   </widget>
   <widget class="QTableView" name="table_fparams">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>490</y>
      <width>1001</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="button_start_preprocess">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>700</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Preprocess</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_delete_row">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>700</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete Row 
 (Highlight Whole Row)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_duplicate_param">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>700</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Duplicate Value 
 Across Rows</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_clear_table">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>700</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear Table</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1049</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
